src/core/timeStepIteration.m

```diff
@@ -219,26 +220,30 @@
 
   Uk( neumdofs(oddNeumDofsInds ) ) = Uk( neumdofs(oddNeumDofsInds ) ) + deltaured(oddNeumDofsInds ) ;
 
-  nNodes = length( Uk) / 6 ;
+  % nNodes = length( Uk) / 6 ;
 
   deltauComplete = zeros( size( Uk)) ;
   deltauComplete( neumdofs ) = deltaured ;
+  nodeAngDofs=2:2:length(Uk);
 
-  for i=1:nNodes
-    nodeDofs = nodes2dofs( i , 6 ) ;
-    nodeAngDofs = nodeDofs(2:2:6)  ;
+  Uk(nodeAngDofs) = deltauComplete(nodeAngDofs) + Uk(nodeAngDofs) ;
```

Comment:
this is a huge optimization since it is a big vectorization... what do you think on this @mvanzulli ? I suggest directly deleting the lines instead of leaving them for the future. we always have the history...

Reply:
Yesss, totally agree! Since we update the angles and displacements with the same formula it has no sense to loop over the nodes. Amazing, this should significantly increase performance (we avoid looping all the nodes at each disp iteration). Congrats @JoaquinViera  and @jorgepz  💯

Reply:
I have added a reference to this PR to just delete deprecated line codes for the angle updates in this issue: https://github.com/ONSAS/ONSAS.m/issues/117.

Reply:
done in [5f3299a](https://github.com/ONSAS/ONSAS.m/pull/648/commits/5f3299a2e547d39c6e569457db8ca790717bc1cc)